### ISSUE TITLE
Add Docker Image Build to CI 

### ISSUE THREAD

**Issue by nouseforaname** [cf-platform-eng/kibosh#47]
Hi,

I wan't to run Kibosh in an Environment where BOSH isn't used. I looked at you CI and it shouldn't be much to also add an Image Build and Publish.. Eventually this could just become a helm chart for comfy deployments on Kube...

If you're interested in having that, let me know your thoughts about this and I'll see to provide a PR shortly..

Best

**Comment by cholick** [comment]
Hey Konstantin, thanks for the offer to PR. Could you elaborate a bit about how this would work? How are you envisioning that the Kibosh image / chart would configure the chart(s) that it offered?

**Comment by nouseforaname** [comment]
I'm baking it into the docker image for now.. I've been thinking to use a PVC later on. I would also like to PR the automatic deployment of svcat API extensions via kibosh and a similar reconfig you currently do with c brokers when creds are provided. we've been looking at the relevant code and think it should be achievable.

**Comment by cholick** [comment]
Ah, I see, yeah, a persistent volume makes a lot of sense. Yeah, I think that would be pretty useful. @jshah-pivotal, @aaronmeza, @mmambrose, @joeeltgroth, @jkjell any additional thoughts?

**Comment by cholick** [comment]
On the svcat extensions, with that project moving to CRDs, that does make me wonder a bit. I think those API extensions would be very short term, until they've completed the move, at which point it would change. Unless it's just a matter of including an optional subchart in the short term.

**Comment by nouseforaname** [comment]
Yeah, thx for clarifying. Was not aware about the upcoming svcat changes
